src/css/imports/captions.less

```diff
@@ -50,6 +50,11 @@
     // Override vtt.js font-size so text resizes as the player size changes
     font-size: inherit;
     line-height: 1.3em;
+    background: transparent;
+}
+
+.jw-text-track-container {
```

Comment:
I just noticed that this element does not have the .jw-reset class added to. @egreaves is there a reason for this? Every element in the player has this class to prevent page styles from affecting player elements.